parts/linux/cloud-init/artifacts/components.json

```diff
@@ -381,6 +381,29 @@
       "multiArchVersions": [
         "v0.7.1"
       ]
+    },
+    {
+      "downloadURL": "mcr.microsoft.com/oss/kubernetes/kube-proxy:*",
```

Comment:
doesn't seem like we're actually using this during VHD builds (install-dependencies.sh), are we not supposed to be caching this anymore?

Reply:
confirmed offline that we still aim to cache this, should probably wait on merging this until after @Devinwong's change merge into master

Reply:
Wouldn't this loop https://github.com/Azure/AgentBaker/pull/4556/files#diff-d6aeb8e845af4c32251915a17dc7c887d73cd628262e83118b9dcd65f7728c98L312-R343 cover the added json entry under 'containerImages' key? I couldn't properly check b/c test pipeline was failing for other reasons, but I'll confirm once that's unblocked. I can also just pause until schema's finalized.

Reply:
ah, yes you're right :)

approved, up to you whether you wanna merge now before Devin's changes or wait until the schema is updated